src/views/index.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha/js/bootstrap.min.js"></script>
+  <link rel="shortcut icon" href="https://dd7g6vmdl9i96.cloudfront.net/a1h0yvvqyf/images/favicon_128x128.png">
```

Comment:
Then let's hold on this for now, I can make a task that uploads the assets to S3 but I think it's not a priority and also the favicon needs to be included in the Html of the different projects anyways.

My bad, I just wanted to get rid of the favicon error in the console